CONTRIBUTING.md

```diff
@@ -127,6 +127,21 @@ Date:   Wed Sep 18 11:44:40 2015 -0700
 ------------------------------------------------------------------------
 ```
 
+### Gems
+
+When making a PR, if you need to add or remove gems, it should be done using the `--conservative` flag:
+```shell
+gem install [gem_name] --conservative
+gem update [gem_name] --conservative
+```
+
+If a manual PR updates the `Gemfile.lock`, please include the output of your `gem update` or `gem install` command in the PR.
```

Comment:
This implies there's some difference between "manual" PRs and other PRs. The "manual" from our notes was referring to doing a PR as opposed to doing large-scale gem updates, it was super confusing. So I would change this to something like:

```suggestion
If your PR includes such an update to `Gemfile.lock`, please include the output of your `gem update` or `gem install` command in the PR.
```

Reply:
It was also partly just me mentally distinguishing an update to `Gemfile.lock` vs. a dependbot bump, but "manual" is redundant in that case because it wouldn't be a PR from a human.